administration/src/Router.tsx

```diff
@@ -60,6 +61,7 @@ const Router = () => {
             { path: '/antrag-einsehen/:accessKey', element: <ApplicationApplicantController /> },
           ]
         : []),
+      ...(projectConfig.selfServiceEnabled ? [{ path: '/erstellen', element: <CardSelfServiceView /> }] : []),
```

Comment:
maybe there are some ideas for a better route name

Reply:
Good point.
The issue here is that we then have on production
`koblenz.sozialpass.app/koblenz/erstellen` which makes no sense

the `projectId` can not be hardcoded in the projectConfig since it is created by the backend, but would have to be retrieved from the api which is difficult because we need to call the api before executing the router.

But you are right, we might have to discuss this